src/components/editor/EditorAreaMenu.tsx

```diff
@@ -71,6 +71,27 @@ export default function Banner(props: { [x: string]: any }) {
 		}
 	};
 
+	const handleHTMLSave = (htmlToSave: string | undefined) => {
+		if (htmlToSave) {
+			const blob = new Blob([htmlToSave], { type: "text/html" });
+	
+			const fileName = "ed-eater.html";
+	
+			const url = URL.createObjectURL(blob);
+
+			const a = document.createElement("a");
+			a.href = url;
+			a.download = fileName;
+			document.body.appendChild(a);
+			a.click();
+
+			URL.revokeObjectURL(url);
+			document.body.removeChild(a);
+		} else {
+			alert("There is no content to save as HTML.");
```

Comment:
when are we expecting such alerts? Can you give a case?

Reply:
When the editor is cleared out(with no content), the alert pops up because the inner HTML is entirely empty. It doesn't contain the template HTML (header, body..) code. So, I thought an alert was preferable. Do you think a blank HTML file should be downloaded instead?

Reply:
Resolved with - Allow users to download empty files